kernel/examples/read-table-multi-threaded/src/main.rs

```diff
@@ -111,13 +110,13 @@ fn send_scan_file(
     size: i64,
     _stats: Option<Stats>,
     dv_info: DvInfo,
-    _transform: Option<ExpressionRef>,
-    partition_values: HashMap<String, String>,
+    transform: Option<ExpressionRef>,
+    _: HashMap<String, String>,
```

Comment:
_should_ we be able to remove this or does this still prove useful elsewhere? (I know there are a number of spots where we use this callback and pass partition values - curious if we will eventually migrate everything to transform or if this is still actually needed)

Reply:
We could. My idea was to keep it for now so existing systems could migrate more slowly. Basically, deprecate it